solidity/dashboard/src/components/SideMenu.jsx

```diff
@@ -1,35 +1,72 @@
 import React from "react"
-import { Link, useRouteMatch } from "react-router-dom"
+import { Link, NavLink, useRouteMatch } from "react-router-dom"
 import * as Icons from "./Icons"
+import { isEmptyArray } from "../utils/array.utils"
+import Divider from "./Divider"
+
+const styles = {
+  overviewDivider: { margin: "1rem 1.5rem" },
+}
 
 export const SideMenu = (props) => {
   return (
     <nav className="side-menu--active">
       <ul className="side-menu__list">
-        <NavLink to="/overview" label="overview" icon={<Icons.Home />} />
-        <NavLink
-          to="/delegation"
-          label="delegation"
-          icon={<Icons.FeesVector />}
+        <li className="side-menu__route-wrapper">
+          <NavLink
+            to="/overview"
+            className="side-menu__route"
+            activeClassName="side-menu__route--active"
+          >
+            Overview
+          </NavLink>
+          <Divider style={styles.overviewDivider} />
```

Comment:
I would encourage taking a look at this divider component with appropriate styling on the `li`. If not, I would consider dropping the intermediate component altogether, instead preferring to use `hr` directly. IMO the intermediate component doesn't seem to bring anything to the table here other than obscuring the final outcome.